Packs/CommonPlaybooks/Playbooks/playbook-Detonate_URL_-_Generic.yml

```diff
@@ -433,345 +460,618 @@ tasks:
+      wait: 1
+      max: 100
+    view: |-
+      {
+        "position": {
+          "x": 4710,
+          "y": 190
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "25":
+    id: "25"
+    taskid: 5535b892-f1b1-42d6-8f7e-c673377dff5c
+    type: playbook
+    task:
+      id: 5535b892-f1b1-42d6-8f7e-c673377dff5c
+      version: -1
+      name: Detonate URL - Hybrid Analysis
+      playbookName: Detonate URL - Hybrid Analysis
+      type: playbook
+      iscommand: false
+      brand: ""
+      description: ""
+    nexttasks:
+      "#none#":
+        - "3"
+    separatecontext: true
+    view: |-
+      {
+        "position": {
+          "x": 5120,
+          "y": 195
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "26":
+    id: "26"
+    taskid: 538c5113-5469-4692-89b2-c1570b74aaef
+    type: playbook
+    task:
+      id: 538c5113-5469-4692-89b2-c1570b74aaef
+      version: -1
+      name: Detonate URL - FireEye AX
+      playbookName: Detonate URL - FireEye AX
+      type: playbook
+      iscommand: false
+      brand: ""
+      description: ""
+    nexttasks:
+      "#none#":
+        - "3"
+    separatecontext: true
+    view: |-
+      {
+        "position": {
+          "x": 5520,
+          "y": 195
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "27":
+    id: "27"
+    taskid: 13bebc22-497e-48e6-85f0-41f6ee62fef1
+    type: playbook
+    task:
+      id: 13bebc22-497e-48e6-85f0-41f6ee62fef1
+      version: -1
+      name: Detonate URL - Hatching Triage
+      playbookName: Detonate URL - Hatching Triage
+      type: playbook
+      iscommand: false
+      brand: ""
+      description: ""
+    nexttasks:
+      "#none#":
+        - "3"
+    separatecontext: true
+    view: |-
+      {
+        "position": {
+          "x": 5920,
+          "y": 195
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
 view: |-
   {
     "linkLabelsPosition": {},
     "paper": {
       "dimensions": {
         "height": 385,
-        "width": 3840,
+        "width": 6690,
         "x": -390,
         "y": 50
       }
     }
   }
 inputs:
-- key: URL
-  value:
-    complex:
-      root: URL
-  required: false
-  description: The URL object of the URL to be detonated.
-  playbookInputQuery:
+  - key: URL
+    value:
+      complex:
+        root: URL
+    required: false
+    description: The URL object of the URL to be detonated.
+    playbookInputQuery:
 outputs:
```

Comment:
Any outputs of the subplaybooks should be added as the outputs of the main parent playbook too